wasm/Driver.cpp

```diff
@@ -523,6 +524,11 @@ void LinkerDriver::link(ArrayRef<const char *> ArgsArr) {
   // Do size optimizations: garbage collection
   markLive();
 
-  // Write the result to the file.
-  writeResult(true);
+  // Write the result to the file
```

Comment:
I think you can suppress ABI file generation by modifying the next link:
https://github.com/EOSIO/lld/blob/disable-abigen/wasm/Writer.cpp#L103-L105

``` c++
  void writeABI() {
     //if (abis.empty())
     if (Config->NoAbigen || abis.empty())
        return;
```